collections/_infographics/emissions/emissions-eu-trends.md

```diff
@@ -13,4 +13,38 @@ data-orig:
   - [ "Eurostat", "https://ec.europa.eu/eurostat/web/products-datasets/-/ENV_AIR_GGE" ]
 ---
 
-{% include includes-local/comment-placeholder.md slug="infografiky/emise-eu-vyvoj" %}
+## How to read this graph
+
+* All emission values are given in units of <glossary id="co2eq">CO<sub>2</sub>eq</glossary>.
+* The infographic shows <glossary id="anthropogenicgreenhousegases">anthropogenic CO<sub>2</sub> emissions</glossary> and emissions of N<sub>2</sub>O, CH<sub>4</sub>, HFC, PFC, SF<sub>6</sub> and NF<sub>3</sub> expressed in units of CO<sub>2</sub> equivalent (CO<sub>2</sub>eq). This unit takes into account the long-term effect of greenhouse gases in the atmosphere and converts them into an amount of CO<sub>2</sub> that would have the same warming effect. For more, see the article on [Global warming potential](https://en.wikipedia.org/wiki/Global_warming_potential).
+* The left part of the graph shows the absolute emissions of the European Union (these are emissions of the EU-27, i.e. countries that were EU members by 2021) in units of CO<sub>2</sub>eq over the last three decades. The different areas of human activity are distinguished by colour. On the right-hand side, the relative change in emissions compared to 1990 is indicated for each sector.
```

Comment:
change "emissions of the European Union (these are emissions of the EU-27, i.e. countries that were EU members by 2021)" to "emissions of the European Union (more precisely, the EU-27, i.e. the countries that were EU members by 2021)" to avoid having emissions twice and a full subclause sentence - slightly cleaner.

Reply:
Thanks, I've updated the text.